Makefile/Makefile.sublime-syntax

```diff
@@ -603,16 +603,33 @@ contexts:
           pop: 1
 
   variable-sub-common:
```

Comment:
as we are using `substitution` written out in full for some context names, I believe it makes sense to be consistent here as well:

```suggestion
  variable-substitution-common:
```

Reply:
You are absolutely right.